ReactiveCocoa/Swift/Flatten.swift

```diff
@@ -59,11 +59,11 @@ extension SignalType where Value: SignalProducerType, Error == NoError {
 	/// Flattens the inner producers sent upon `signal` (into a single signal of
 	/// values), according to the semantics of the given strategy.
 	///
-	/// If an active inner producer fails, the returned signal will forward that
-	/// failure immediately.
-	///
-	/// `Interrupted` events on inner producers will be treated like `Completed`
-	/// events on inner producers.
+	/// - parameter strategy: Strategy used when flattening signals.
```

Comment:
This breaks the rule about using `- parameters:` in the documentation documentation. ☺️